public/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Hiba's Hungry Angels</title>
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+  <header>
+    <h1>HIBA FEEDS</h1>
+  </header>
+
+  <!-- buttons -->
+  <section>
+    <div class="button-wrap">
+      <button class="tag-button">All</button>
```

Comment:
could you add "data-tag" attribute (equal to "all", "js" etc.) to these buttons so we can use them in js, please.